### ISSUE TITLE
Modify the Site

### ISSUE THREAD

**Issue by Sastha-Kumar-N** [Bharat-Genome-Database-BGDB/SSF#12]
The site have various styling and alignment issues, we need to track them down here as list of things to do and make it error free

**Comment by shreyas-a-s** [comment]
Alignment issues means in mobile screens or in all screens?

**Comment by Sastha-Kumar-N** [comment]
> Alignment issues means in mobile screens or in all screens?

Nope, overall fit and finish of the website feels a little non standard, have to make subtle changes that will reorder them, like having spaces on both sides of the content, proper headings and colour formatting, (H1, H2 headings etc..) should be used.

For now, mark all these issues by reviewing them

**Comment by shreyas-a-s** [comment]
I think we should talk over a call. That way we can discuss a bit more. After that we can update the issue section.
